corehq/apps/accounting/templates/accounting/subscriptions.html

```diff
@@ -5,7 +5,6 @@
 
 {% block js %}{{ block.super }}
     <script src="{% static 'accounting/js/stay-on-tab.js' %}"></script>
-    <script src="{% static 'accounting/js/accounting.billing_info_handler.js' %}"></script>
```

Comment:
I did mean to remove this. It was causing an error because the same thing is being loaded in the base template, and hqDefine doesn't allow the same import to be defined twice